### PR TITLE
improve currency and align it to center

### PR DESCRIPTION
**Pull request by bebnev-pro**

### CODE REVIEW

**Comment by bebnev-pro** [comment]
image
is it looks better?

File: index.html [hunk 1]
```diff
@@ -9,10 +9,12 @@
   </head>
   <body>
     <div id="sidebar" class="sidebar">
-      <button class="select-currency-btn" data-currency-id="USD">USD</button>
-      <button class="select-currency-btn" data-currency-id="EUR">EUR</button>
-      <button class="select-currency-btn" data-currency-id="JPY">JPY</button>
-      <button class="select-currency-btn" data-currency-id="GBP">GBP</button>
+      <div class="currency-wrapper">
```
Comment (9brada6) [inline index.html:12]: I agree that a wrapper of the buttons is needed.

File: src/index.css [hunk 2]
```diff
@@ -118,3 +119,13 @@ body {
   background-color: #b30000;
   color: white;
 }
+
+.currency-wrapper {
```
Comment (9brada6) [inline src/index.css:123]: I don't like the position absolute, while it works, that's not how you align the items. There is no reason for the currency wrapper to be positioned absolute in your display. If you want to fix this, use flex on the sidebar:

.sidebar {
    display: flex;
    align-items: center;
}

**Review by 9brada6 — CHANGES_REQUESTED** [review]
In css, change add only the following:

.sidebar{
  display: flex;
  align-item: center;
}

 and remove the absolute position. You can style next the buttons how you want.

**Review by 9brada6 — APPROVED** [review]

**Comment by 9brada6** [comment]
Merged changes bebnev with the absolute positioning of the buttons.

**Pull request merged**